package.json

```diff
@@ -48,6 +48,7 @@
     "eslint": "3.18.0",
     "eslint-plugin-google-camelcase": "0.0.2",
     "express": "4.14.0",
+    "fetch-mock": "^5.12.1",
```

Comment:
Can you also update `yarn.lock` by running `yarn upgrade --save-dev` and including the diffs in this PR?

/cc @erwinmombay